mbbsd/bbs.c

```diff
@@ -433,6 +433,38 @@ int IsFreeBoardName(const char *brdname)
     return 0;
 }
 
+/*
+ * Check if user phone number is verified by sms
+ *
+ * This function will check is current user have already register their
```

Comment:
`will` should be redundant.

`will` makes the comment describe what will be done when the function is called instead of what the function per se does.